src/components/NcAppNavigationItem/NcAppNavigationItem.vue

```diff
@@ -771,208 +771,6 @@ export default {
-		z-index: 100; /* above the bullet to allow click*/
-		display: flex;
-		overflow: hidden;
-		flex: 1 1 0;
-		box-sizing: border-box;
-		min-height: $clickable-area;
-		padding: 0;
-		white-space: nowrap;
-		color: var(--color-main-text);
-		background-repeat: no-repeat;
-		background-position: $icon-margin center;
-		background-size: $icon-size $icon-size;
-		line-height: $clickable-area;
-
-		.app-navigation-entry-icon {
-			display: flex;
-			align-items: center;
-			flex: 0 0 $clickable-area;
-			justify-content: center;
-			width: $clickable-area;
-			height: $clickable-area;
-			background-size: $icon-size $icon-size;
-			background-repeat: no-repeat;
-			background-position: $icon-margin center;
-		}
-
-		.app-navigation-entry__name {
-			overflow: hidden;
-			max-width: 100%;
-			white-space: nowrap;
-			text-overflow: ellipsis;
-		}
-
-		.editingContainer {
-			width: calc(100% - #{$clickable-area});
-			margin: auto;
-		}
-	}
-}
-/* Second level nesting for lists */
-.app-navigation-entry__children {
-	position: relative;
-	display: flex;
-	flex: 0 1 auto;
-	flex-direction: column;
-	width: 100%;
-	gap: var(--default-grid-baseline, 4px);
-
-	.app-navigation-entry {
-		display: inline-flex;
-		flex-wrap: wrap;
-		padding-left: $icon-size;
-	}
-}
-
-/* Deleted entries */
-.app-navigation-entry__deleted {
-	display: inline-flex;
-	flex: 1 1 0;
-	padding-left: $clickable-area - $icon-margin !important;
-	.app-navigation-entry__deleted-description {
-		position: relative;
-		overflow: hidden;
-		flex: 1 1 0;
-		white-space: nowrap;
-		text-overflow: ellipsis;
-		line-height: $clickable-area;
-	}
-}
-
-/* counter and actions */
-.app-navigation-entry__utils {
-	display: flex;
-	min-width: $clickable-area;
-	align-items: center;
-	flex: 0 1 auto;
-	justify-content: flex-end;
-	&#{&}--display-actions .action-item.app-navigation-entry__actions {
-		display: inline-block;
-	}
-	/* counter */
-	.app-navigation-entry__counter-wrapper {
-		// Add slightly more space to the right of the counter
-		margin-right: calc(var(--default-grid-baseline) * 3);
-		display: flex;
-		align-items: center;
-		flex: 0 1 auto;
-	}
-	/* actions */
-	.action-item.app-navigation-entry__actions {
-		display: none;
-	}
-}
-
-// STATES
-/* editing state */
-.app-navigation-entry--editing {
-	.app-navigation-entry-edit {
-		z-index: 250;
-		opacity: 1;
-	}
-}
-
-/* deleted state */
-.app-navigation-entry--deleted {
-	.app-navigation-entry-deleted {
-		z-index: 250;
-		transform: translateX(0);
-	}
-}
-
-/* pinned state */
-.app-navigation-entry--pinned {
-	order: 2;
-	margin-top: auto;
-	// only put a marginTop auto to the first one!
-	~ .app-navigation-entry--pinned {
-		margin-top: 0;
-	}
-}
-
-// Add more contrast for active entry
-[data-themes*='highcontrast'] {
-	.app-navigation-entry {
-		&:active {
-			background-color: var(--color-primary-element-light-hover) !important;
-		}
-	}
-}
+<style scoped lang="scss">
+@import '../../assets/NcAppNavigationItem';
```

Comment:
`@import` is deprecated you should use `@use`

Reply:
done

Reply:
I tried this and it didnt work: https://github.com/nextcloud-libraries/nextcloud-vue/actions/runs/6734727700/job/18306263029